server/static/js/actions.js

```diff
@@ -0,0 +1,607 @@
+// actions are triggered by UI events. It mostly involves calling backend API, and as a result, updating the store.
```

Comment:
This file includes the functions interacting with server or drawing graphs originally in `server/static/js/acorn_leaflet.js`.